content/end.html

```diff
@@ -70,51 +75,62 @@ <h3> Inspirations </h3>
 <p class="full">
 Systems Design Engineering 2017 Class Profile
 <a target="_blank" href="http://joeyloi.com/SYDE2017classprofile.pdf">
-http://joeyloi.com/SYDE2017classprofile.pdf
+  http://joeyloi.com/SYDE2017classprofile.pdf
 </a>
 </p>
 
 <p class="full">
-HackerRank 2018 Developer Skills Report
-<a target="_blank" href="http://research.hackerrank.com/developer-skills/2018/">
```

Comment:
Maybe we should also consider adding the 2020 for SE specifically https://uw-se-2020-class-profile.github.io/profile.pdf